NDB.Covid19/NDB.Covid19.Droid/Resources/layout/messages_page.xml

```diff
@@ -61,6 +61,7 @@
             android:textAppearance="@style/PrimaryText"
             android:layout_marginHorizontal="@dimen/defaultHorizontalMargin"
             android:paddingBottom="@dimen/defaultVerticalMargin"
+            android:paddingTop="8dp"
```

Comment:
Maybe make into a dimen since it's used twice, and probably also will be used in consent 3 page?

Reply:
done